src/request/components/request-view.scss

```diff
@@ -376,6 +376,69 @@
     border-bottom-width: 2px;
 }
 
+.master-detail-tri-panel {
+    display: flex;
+    flex-direction: column;
+}
+
+.master-detail-tri-panel-master {
+    flex: 3;
+    min-height: 100px;
+    overflow-y: auto;
+    padding: 0 20px 20px 20px;
+}
+
+.master-detail-tri-panel-detail {
+    display: flex;
+    flex: none;
+    flex-direction: row;
+    height: 250px;
+}
+
+.master-detail-tri-panel-detail-panel {
+    display: flex;
+    flex: none;
+    flex-direction: column;
+    width: 50%;
+}
+
+.master-detail-tri-panel-detail-margin {
+    background: #ccc;
+    flex: none;
+    width: 1px;
+}
+
+.master-detail-tri-panel-detail-panel-title {
+    background-color: #323d42;
+    color: white;
+    font-weight: bold;
+    padding-bottom: 5px;
+    padding-left: 10px;
+    padding-top: 5px;
+}
+
+.master-detail-tri-panel-detail-panel-content {
+    overflow: auto;
+}
+
+.tab-data {
+    display: flex;
+}
+
+.tab-data-operation-count {
+    font-family: 'Segoe UI', 'Selawik Semibold', Tahoma, Geneva, Verdana, sans-serif;
+    font-size: 175%;
+}
+
+.tab-data-operation-table {
+    table-layout: fixed;
+}
+
+.tab-data-operation-table-command-column {
+    overflow: hidden;
+    white-space: nowrap;
+}
+
```

Comment:
Yes, the "master" panel will scroll in that case. (There is a case where, if the browser window is too small, the Client's request details panel will itself also scroll, which is a little weird, but I believe it's due to the styling of the details panel itself.  The Web Services tab behaves the same way.)